catroid/src/androidTest/java/org/catrobat/catroid/common/bluetooth/ConnectionDataLogger.java

```diff
@@ -181,38 +155,10 @@ private ConnectionDataLogger(boolean local) {
 		}
 	}
 
-	private ConnectionDataLogger(DeviceModel deviceModel) {
-		connectionProxy = new LocalConnectionProxy(logger, deviceModel);
-		BluetoothTestUtils.hookInConnection(connectionProxy);
-	}
-
 	public static ConnectionDataLogger createLocalConnectionLogger() {
 		return new ConnectionDataLogger(true);
 	}
 
-	public static ConnectionDataLogger createLocalConnectionLoggerWithTestDevice(BluetoothDevice testDevice) {
-		BluetoothTestUtils.hookInTestDevice(testDevice);
-		return new ConnectionDataLogger(true);
-	}
-
-	public static ConnectionDataLogger createLocalConnectionLoggerWithDeviceModel(DeviceModel deviceModel) {
-		return new ConnectionDataLogger(deviceModel);
-	}
-
-	public static ConnectionDataLogger createLocalConnectionLoggerWithDeviceModelAndTestDevice(DeviceModel deviceModel, BluetoothDevice testDevice) {
-		BluetoothTestUtils.hookInTestDevice(testDevice);
-		return new ConnectionDataLogger(deviceModel);
-	}
-
-	public static ConnectionDataLogger createBluetoothConnectionLogger() {
-		return new ConnectionDataLogger(false);
-	}
-
-	public static ConnectionDataLogger createBluetoothConnectionLoggerWithTestDevice(BluetoothDevice testDevice) {
-		BluetoothTestUtils.hookInTestDevice(testDevice);
-		return new ConnectionDataLogger(false);
-	}
-
 	public void disconnectAndDestroy() {
 		if (connectionProxy != null) {
 			connectionProxy.disconnect();
```

Comment:
LGTM since every dependence is also deleted. Also unused at current develop :+1: